### PR TITLE
Mads/traces diagram

### PR DESCRIPTION
**Pull request by mads-hartmann**
## Description

As part of our Gitpod Tech Talk I made the following SVG which shows an example deployment of Gitpod and Monitoring Satellite with a focus on tracing.

@ArthurSens I'm happy to just leave it in the docs folder, but how to you feel?

## Related Issue(s)

N/A

## Release Notes

```release-note
NONE
```

## Documentation

N/A

### CODE REVIEW

**Comment by ArthurSens** [comment]
Do you want to save the diagram source code in case you need to modify it in the future? That's a fair point to store it here I think :)

Nit: Maybe rephrase the commit messages?

**Comment by mads-hartmann** [comment]
@ArthurSens Yeah, and by linking to the SVG on main Notion and other resources should update if the diagram is changed in the future ☺️ I'll squash and merge and write a better message.
